aws/aws_aurora/cluster/main.tf

```diff
@@ -0,0 +1,46 @@
+# Summary: Create an RDS Aurora MySQL 1 Cluster instance that does not inherit configuration from the cluster
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 0.14.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.38"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+provider "aws" {
+  region  = "us-east-1"
+  profile = "terraform-examples"
+  default_tags {
+    tags = {
+      cs_terraform_examples = "aws_aurora"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/values/variables.html
+variable "changeme_aws_aurora_username" {
+  description = "the username for the login credentials to your db"
+  default     = "username_example"
+}
+
+# Documentation: https://www.terraform.io/docs/language/values/variables.html
+variable "changeme_aws_aurora_password" {
+  description = "the password, which must be more than 8 characters long"
+  default     = "password_example"
+}
+
+# Documentation: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster
+resource "aws_rds_cluster" "changeme_aws_aurora_cluster" {
+  # The login credentials are stored as plain-text in the state files
+  # https://www.terraform.io/docs/language/state/sensitive-data.html
+  # For security, use a secrets manager
+  # https://registry.terraform.io/modules/yurymkomarov/rds-aurora-cluster/aws/latest#secretsmanager_secret
+  master_username     = var.changeme_aws_aurora_username
+  master_password     = var.changeme_aws_aurora_password
+  skip_final_snapshot = true
+}
```

Comment:
This resource caused a error with `terraform apply` done by CI:
- https://github.com/ContainerSolutions/terraform-examples/runs/3229040694?check_suite_focus=true

```
Plan: 1 to add, 0 to change, 0 to destroy.
aws_rds_cluster.changeme_aws_aurora_cluster: Creating...
╷
│ Error: error creating RDS cluster: AccessDenied: User: arn:aws:iam::684157722011:user/terraform-examples is not authorized to perform: rds:CreateDBCluster on resource: arn:aws:rds:us-east-1:684157722011:cluster:tf-20210803085818971100000001
│ 	status code: 403, request id: d380fcf2-c6e7-47c7-ae23-dacc3f0d6782
│ 
│   with aws_rds_cluster.changeme_aws_aurora_cluster,
│   on main.tf line 38, in resource "aws_rds_cluster" "changeme_aws_aurora_cluster":
│   38: resource "aws_rds_cluster" "changeme_aws_aurora_cluster" ***
│ 
╵
/home/runner/work/terraform-examples/terraform-examples/aws/aws_aurora/cluster
```

Seems to be related to some missing permissions.